api/models/config_keys.py

```diff
@@ -15,7 +16,9 @@ class Config:
 
 
 class AppConfig(SnakeModel):
-    VideoPath: Optional[str] = Field(None, example='/repo/data/TownCentreXVID.avi')
+    VideoPath: Optional[str] = Field(None, example='/repo/data/gard1-4.mp4')
+    Host: Optional[str] = Field(None, example='0.0.0.0')
+    Port: Optional[str] = Field(None, example='8001')
```

Comment:
I think we should leave 8000 as example
```suggestion
    Port: Optional[str] = Field(None, example='8000')
```

Reply:
Actually, host and port are in ApiConfig so should be removed from here. or from ApiConfig